core/org.wso2.carbon.server/src/main/java/org/wso2/carbon/server/extensions/DropinsBundleDeployer.java

```diff
@@ -50,6 +50,7 @@
 
     private static String  dropinsDirPath;
     private static String bundlesInfoDirPath;
+    private static final String DEFAULT_BUNDLE_VERSION = "0.0.0";
```

Comment:
Since usage of this string is only within getNewBundleInfoLines private method, better to define this constant over there, WDYT?